composer.json

```diff
@@ -27,6 +27,7 @@
     "ext-json": "*",
     "contributte/psr7-http-message": "~0.7.0",
     "doctrine/annotations": "^1.8.0",
+    "koriym/attributes": "^1.0",
```

Comment:
I prefer not to add new dependencies to project, but this one allows us remove lot of custom boiler plate code (I actually wrote it manually first creating lot of code that can break :)).

It adds doctrine-interface-based Reader, that undrestands both attributes and annotations. See `DualReader` class.

Reply:
I don't like it either. But I understand it. We can keep it here for this version and after switching to PHP 8, I will destroy it. :-D